README.md

```diff
@@ -25,6 +25,19 @@ This repository specifically contains these PDS registry application resources:
 All users and developers of the NASA-PDS software are expected to abide by our [Code of Conduct](https://github.com/NASA-PDS/.github/blob/main/CODE_OF_CONDUCT.md). Please read this to ensure you understand the expectations of our community.
 
 
+## Utilities
+* Titan Treks
+    * The Titan Treks utilities can be used to create pds4 labels for the layers given in the Titan Treks API
```

Comment:
You should add a line for users to deploy the package:
In the cloned repository:
`pip install .` for users 
`pip install -e '.[dev]'` for developers

Reply:
This package will be hosted [at the "cheeseshop"](https://pypi.org/), right? In that case:

    pip install pds.registry